File: app/src/main/java/com/xjx/helper/ui/home/activity/tod/customview/DrawableTextViewActivity.java
package com.xjx.helper.ui.home.activity.tod.customview;

import com.xjx.helper.R;
import com.xjx.helper.base.CommonBaseTitleActivity;
import com.xjx.helper.enums.PlaceholderStatus;

public class DrawableTextViewActivity extends CommonBaseTitleActivity {

    @Override
    protected int getTitleLayout() {
        return R.layout.activity_drawable_text_view2;
    }

    @Override
    protected void initData() {
        super.initData();
        setTitleContent("DrawableTextView");
        SwitchLoadingStatus(PlaceholderStatus.NONE);

    }
}

File: app/src/main/java/com/xjx/helper/ui/home/activity/tod/DrawableTextViewActivity.kt
package com.xjx.helper.ui.home.activity.tod

import com.xjx.helper.R
import com.xjx.helper.base.CommonBaseTitleActivity
import com.xjx.helper.enums.PlaceholderStatus

class DrawableTextViewActivity : CommonBaseTitleActivity() {

    override fun getTitleLayout(): Int {
        return R.layout.activity_drawable_text_view
    }

    override fun initView() {
        super.initView()
        SwitchLoadingStatus(PlaceholderStatus.NONE)

    }
}
